packages/botonic-react/src/components/button.jsx

```diff
@@ -149,14 +166,26 @@ export const Button = props => {
           color: buttonTextColor,
           backgroundColor: buttonBgColor,
           ...(props.disabled && autoDisable && disabledStyle),
+          position: 'relative',
         }}
         bottom={props.bottomRadius}
       >
         {props.children}
+        {props.url && urlIcon && (
+          <img
+            className='botonic-url-icon'
+            style={{ width: '20px', position: 'absolute', right: '12px' }}
+            src={resolveImage(urlIcon)}
+          />
+        )}
       </StyledButton>
     )
   }
 
+  const propertyDisabled = value => {
+    return value === false
+  }
+
```

Comment:
remove this function

Reply:
I'm using this function to check if the property is set to false and even if there is an image, not showing the image (but maybe this can be simplified an just let the image be shown if there is image and enable rop is set to false.)

Reply:
I think it's messed up, you're using this function to check for false, and when it returns true in the ternary where it's used you return undefined...